extra/utils/validation.js

```diff
@@ -0,0 +1,42 @@
+const users =
+
+    [
+        {
+        traineeEmail: 'neha.goel@successive.tech',
+        reviewerEmail: 'megha.raw@successive.tech',
+        },
+        {
+        traineeEmail: 'shivam.sharma@sucssive.tech',
+        reviewerEmail: 'meghaa.rawat@successive.tech',
+        },
+        {
+        traineeEmail: 'nehaa@successive.tech',
+        reviewerEmail: 'megha.r@suessive.tech',
+        }
+    ];
+valid=[];
+invalid=[]; 
+function validateEmail(x)
+{
+   const regex =  /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((successive.tech))$/;
+   return regex.test(x);
+
+}
+function validateUsers(users)
+{
+    users.forEach((element) => {const {traineeEmail, reviewerEmail} = element;
+     if(validateEmail(traineeEmail))
+	valid.push(traineeEmail);
```

Comment:
Check the data of traineeEmail and reviewerEmail simultaneously.  One object of the array as combined.